### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by le4ker**
Add tflint (that was added [here](https://github.com/mfussenegger/nvim-lint/pull/305)) in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,6 @@ git clone \
 - If using [vim-plug][3]: `Plug 'mfussenegger/nvim-lint'`
 - If using [packer.nvim][4]: `use 'mfussenegger/nvim-lint'`
 
-
```
Comment (mfussenegger) [inline README.md:36]: Could you please revert the unrelated formatting changes?

Comment (le4ker) [reply README.md]: sure, I reverted them
